src/Provider/AbstractProvider.php

```diff
@@ -24,16 +24,6 @@
     const ACCESS_TOKEN_UID = null;
 
     /**
-     * @var string HTTP method used to fetch access tokens.
```

Comment:
I removed these because they are already constants as they stand. I like that we can accommodate any request method now, which makes these feel noisy.